internal/app/container.go

```diff
@@ -0,0 +1,60 @@
+package app
+
+import (
+	"github.com/shulganew/gophermart/internal/config"
+	"github.com/shulganew/gophermart/internal/ports/client"
+	"github.com/shulganew/gophermart/internal/ports/storage"
+	"github.com/shulganew/gophermart/internal/services"
+)
+
+// A container pattern.
+type Application struct {
+	stor     *storage.Repo
+	calcSrv  *services.CalculationService
+	client   *client.Accrual
+	userSrv  *services.UserService
+	accSrv   *services.AccrualService
+	orderSrv *services.OrderService
+	conf     *config.Config
+}
+
+func NewApp(conf *config.Config, stor *storage.Repo) *Application {
+	application := &Application{}
+	application.conf = conf
+	application.calcSrv = services.NewCalcService(stor)
+	application.userSrv = services.NewUserService(stor)
+	application.client = client.NewAccrualClient(conf)
+	application.accSrv = services.NewAccrualService(stor, application.client)
+	application.orderSrv = services.NewOrderService(stor)
+	application.stor = stor
+
+	return application
+}
+
+func (c *Application) GetCalculationService() *services.CalculationService {
```

Comment:
я сам ввел в заблуждение - в go не принято добавлять get, т.е. просто обращаемся как к полю структуры
```suggestion
func (c *Application) CalculationService() *services.CalculationService {
```